homework/yuliya-yavorskaia/lesson-10/01/less/style.less

```diff
@@ -0,0 +1,245 @@
+// out: ../css/style.css
+
+@font-face {
+  font-family: 'Raleway';
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+
+  src: local(''),
+    url('../fonts/raleway-v16-latin-300.woff2') format('woff2'),
+    url('../fonts/raleway-v16-latin-300.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Raleway';
+  font-style: normal;
+  font-weight: 700;
+  font-display: swap;
+
+  src: local(''),
+    url('../fonts/raleway-v16-latin-700.woff2') format('woff2'),
+    url('../fonts/raleway-v16-latin-700.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Raleway';
+  font-style: normal;
+  font-weight: 800;
+  font-display: swap;
+
+  src: local(''),
+    url('../fonts/raleway-v16-latin-900.woff2') format('woff2'),
+    url('../fonts/raleway-v16-latin-900.woff') format('woff');
+}
+
+body {
+  width: auto;
+  min-width: 320px;
+  margin: 0;
+  padding: 0;
+
+  font-family: 'Raleway', 'Arial', sans-serif;
+  font-weight: 300;
+
+  background: url(../img/bg.jpg);
+}
+
+a {
+  display: block;
+
+  text-decoration: none;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+}
+
+.container {
+  max-width: 1180px;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.page-header__top-wrapper {
+  display: flex;
+
+  padding: 50px 0 102px 0;
+}
+
+.page-header__logo {
+  &:hover {
+    opacity: 0.5;
+  }
+
+  &:focus,
+  &:active {
+    opacity: 0.3;
+  }
+}
+
+.main-nav {
+  width: 48%;
+  margin-left: auto;
+
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 25px;
+  text-transform: uppercase;
+}
+
+.site-list {
+  display: flex;
+  justify-content: space-between;
+
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+}
+
+.site-list__item a {
+  padding-top: 13px;
+  padding-bottom: 13px;
+
+  color: #000000;
+
+  &:hover {
+    color: #ff6033;
+  }
+
+  &:focus,
+  &:active {
+    color: #dad0d0;
+  }
+}
+
+.site-list__item--active a {
+  position: relative;
+
+  margin-left: 5px;
+}
+
+.site-list__item--active a::before {
+  content: "";
+  position: absolute;
+  top: 8px;
+  left: -1px;
+
+  width: 50px;
+  height: 3px;
+  background-color: #ff6033;
+}
+
+.intro {
+  position: relative;
+  width: 100%;
+  padding: 150px 0 400px 0;
```

Comment:
there is clearly something wrong with these indents, the values should not be so large